UKHomeOffice/sgmr-service
# SPCR-208 Fix crown copyright link

## Description
The crown copyright link previously didn't work. This has now been fixed.

## To Test
- Go to any page on the service which has a footer
- Click on the crown copyright link
- Your browser should redirect to https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
